points/SO3/LICENSE

```diff
@@ -0,0 +1,11 @@
+The SO3 points clouds have been generated from the ISOI software available
+at this webpage : 
+https://mitchell-web.ornl.gov/SOI/index.php
+
+This software is an implementation of the two following papers:
```

Comment:
Doesn't really feel like a license. A README maybe?

Reply:
I removed the 2 papers information from the LICENSE file on [6648f2d](https://github.com/GUDHI/gudhi-data/pull/11/commits/6648f2d633ff8823066161de005074b9672bbee3)